fqlx/queries/function.js

```diff
@@ -0,0 +1,28 @@
+const { fql } = require('fauna')
+const { and } = require('./utils')
+
+/**
+ * Creates or updates a function according to the passed parameters.
+ *
+ * @param params The params to pass to the create or update call.
+ * @returns An FQL X Query
+ */
+const createUpdateFunction = (params) => {
+  return fql`
+  let results = if (Function.byName(${params.name}) != null) {
```

Comment:
The `!= null` will break somewhat soon, once we land support for named null documents. I don't think there's any action here, just something to lookout for.